style.css

```diff
@@ -90,6 +90,10 @@
     .introduction{
         margin-right: 106px;
     }
+    .intro_img{
+        width: 243px;
+        height: 243px;
+    }
     .intro_head{
         font-size: 48px;
```

Comment:
These two below sections are not matching. Same goes for the below two blocks

![Screenshot 2023-03-13 at 2 46 32 PM](https://user-images.githubusercontent.com/38748298/224658026-e32f7031-250a-498e-903f-6d9fe17c724a.png)


![Screenshot 2023-03-13 at 2 46 26 PM](https://user-images.githubusercontent.com/38748298/224658018-1eff050c-dc44-4817-8934-2e203fbe7723.png)